package.json

```diff
@@ -21,7 +21,8 @@
     "phantomjs-prebuilt": "^2.1.16",
     "psc-package": "^3.0.1",
     "pulp": "15.0.0",
-    "puppeteer": "^5.4.1"
+    "puppeteer": "^5.4.1",
+    "purescript": "^0.13.8"
```

Comment:
Doesn't build on newer versions it seems.

Reply:
NB: this is not needed, we use a postinstall script for this. https://github.com/restaumatic/purescript-specular/pull/73/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R7